packages/manager-overthebox/src/details/overTheBox-details.constant.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+
+export default {
```

Comment:
overTheBox-details.constant**s**.js

Maybe we should export each constant separately and export all by default : 

```js
export const CHART = { 
// ...
}

export default { CHART };
```